doc/advanced_configuration.rst

```diff
@@ -481,7 +481,7 @@ Binder links will point to these notebooks.
    Binder link, these files will be used to create the environment.
    For more information on what files you can use, see `preparing your
    repository <https://mybinder.readthedocs.io/en/latest/using.html#preparing-a-repository-for-binder>`_
-   in the `Binder documentation <docs.mybinder.org>`_ for more information on
+   in the `Binder documentation <docs.mybinder.org>`__ for more information on
```

Comment:
I'm wondering why you need to change the underscore? Trying to figure out(still ignorant), I realized this link is broken. Without `http` sphinx considers it as a local link.

Reply:
The `_` makes it a link that can be used elsewhere, and it's named the same way so it creates a warning. The alternative would be just to not include the link in the second one, but this way makes it so that if one is removed both still work

Reply:
See e.g.

http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks